composer.json

```diff
@@ -8,7 +8,8 @@
     "homepage": "https://laminas.dev",
     "license": "BSD-3-Clause",
     "require": {
-        "php": "^7.3 || ~8.0.0",
+        "php": "^7.3|^8.0",
```

Comment:
Should be `^7.4 || ~8.0.0 || ~8.1.0`

Reply:
@Ocramius aren't you planning to keep it maintained after 8.1? Seems a bit redundant to keep adding minor versions to compatibility entry?

Reply:
We just don't know how PHP 8.2 will break things: remember that PHP does not follow SemVer, and PHP 8.1 has already been quite brutal on a huge amount of our packages.

Reply:
@Ocramius no problem - that's now been updated.